src/ui/public/notify/partials/toaster.html

```diff
@@ -6,7 +6,9 @@
         <span ng-show="notif.count > 1" class="badge">{{ notif.count }}</span>
 
         <i class="fa" ng-class="'fa-' + notif.icon" tooltip="{{notif.title}}"></i>
-        <kbn-truncated orig="{{notif.content}}" length="250" class="toast-message" /></kbn-truncated>
+
+        <kbn-truncated ng-if="notif.markdown" source="{{notif.markdown | markdown}}" is-html="true" length="250" class="toast-message" /></kbn-truncated>
```

Comment:
Since you use the markdown filter here, you should also import `public/filters/markdown` in `src/ui/public/notify/notify.js`